dotCMS/src/main/java/com/dotcms/util/CollectionsUtils.java

```diff
@@ -981,6 +981,25 @@
 	    return new ImmutableListCollector<>();
     }
 
+    /**
+     * Returns true if all elements on setA are contained on setB
+     * @param setA Set
+     * @param setB Set
+     * @return boolean
+     */
+    public static <T> boolean containsAllElements(final Set<T> setA,
+                                           final Set<T> setB) {
+
+	    for (final T element : setA) {
```

Comment:
Why do we have this method?  java has this out of the box. `Collection.containsAll(Collection)`

https://docs.oracle.com/javase/8/docs/api/java/util/Collection.html#containsAll-java.util.Collection-

Reply:
I think you right, on it